nvflare/client/api.py

```diff
@@ -99,13 +99,18 @@ def clear():
 def system_info() -> Dict:
     """Gets NVFlare system information.
 
+    System information will be available after a valid FLModel is received.
```

Comment:
Is this still true? Your PR comment states "Fix "system_info" api, it will now work before any train/evaluation is done"

Reply:
This is still True, the way I fix the "system_info" is to check if there is any FLModel received first.
If NOT then this command will try to receive a FLModel.